include/builtins.h

```diff
@@ -15,10 +15,10 @@
 
 # include <stdbool.h>
 
-int	builtin_echo(char **argv, bool no_newline_flag);
-int	builtin_pwd(char **argv);
-int	builtin_env(char **argv);
-int	builtin_export(char **argv);
-int	builtin_unset(char **argv);
+int	builtin_echo(int argc, char **argv);
+int	builtin_pwd(int argc, char **argv);
+int	builtin_env(int argc, char **argv);
+int	builtin_export(int argc, char **argv);
+int	builtin_unset(int argc, char **argv);
```

Comment:
So that the interface is uniform, and we can give our builtins the same input as normal executables, without pre-processing data. I'm not sure how calling builtins will differ from calling a normal executable in terms of forks/processes, so I suggest to leave it like that for the time being. 
**argv is passed into execve() with argv[0]=executable_name, so this part will be unchanged.
We can always remove argc in the end if we don't heed it, and since it's always (void)argc, we don't really risk breaking our program or creating new bugs by removing it later.